src/utils/update-job.js

```diff
@@ -0,0 +1,12 @@
+export const updatedTasks = (e) => {
+  let numberOfTasks = e.sorCodesCollection
+    ? (Object.keys(e).length - 1) / 2
+    : Object.keys(e).length / 2
```

Comment:
How come we're dividing by two here? I was a bit confused why that's needed.

Reply:
I'm a bit confused with what is happening here. Maybe worth adding a comment if it's not possible to simplify?

Reply:
to be able to get info on existing tasks
they are saved in formData separately 
`{sor-code-0: "DES5R004 - Emergency call out", quantity-0: 1, sor-code-1: "PLP5R054 - RENEW MIXER TAP", quantity-1: 1, sorCodesCollection: Array(1)}`
sorCodeCollection: Array is all SOR code that were added extra

Reply:
Actually I think there is a better way for that

Reply:
Might be worth having the existing tasks managed in its own collection array as there may be a case where it is multiple sors